core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -429,40 +425,36 @@ public void listStatus() throws Exception {
    */
   @Test
   public void listStatusWithoutMountInfo() throws Exception {
-    FileInfo fileInfo1 = new FileInfo()
-        .setLastModificationTimeMs(111L)
-        .setLastAccessTimeMs(123L)
-        .setFolder(false)
-        .setOwner("user1")
-        .setGroup("group1")
-        .setMode(00755);
-    FileInfo fileInfo2 = new FileInfo()
-        .setLastModificationTimeMs(222L)
-        .setLastAccessTimeMs(234L)
-        .setFolder(true)
-        .setOwner("user2")
-        .setGroup("group2")
-        .setMode(00644);
-
     Path path = new Path("/dir");
-    alluxio.client.file.FileSystem alluxioFs =
-        mock(alluxio.client.file.FileSystem.class);
-    FileSystem alluxioHadoopFs = new FileSystem(alluxioFs);
-    URI uri = URI.create(Constants.HEADER + "host:1");
+    FileSystem alluxioHadoopFs = new FileSystem();
+    FileSystemContext fsContext = PowerMockito.mock(FileSystemContext.class);
+    URI uri = URI.create(Constants.HEADER + "localhost:1");
     Configuration configuration = getConf();
+
+    // exclude mount info
```

Comment:
Now this function tests both including and excluding mount info. Could you split them into two separate methods?